go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/grpc-ecosystem/grpc-health-probe v0.3.6
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.3.0
 	github.com/mikefarah/yq/v3 v3.0.0-20201202084205-8846255d1c37
-	github.com/onsi/ginkgo v1.14.2
-	github.com/onsi/gomega v1.10.3
+	github.com/onsi/ginkgo v1.16.1
```

Comment:
This change came from https://github.com/operator-framework/operator-lifecycle-manager/commit/b79ce0bfb0cd26550c7eb28d72aecb7f4b8e6367 -- could you resolve the conflict in the other direction to keep this fix clean?